types/request-promise-native/index.d.ts

```diff
@@ -1,31 +1,75 @@
-// Type definitions for request-promise-native v1.0.3
+// Type definitions for request-promise-native 1.1
```

Comment:
Correct. The version is for the types, not the library. The linter disallows the "v" prefix and the "patch" number. Didn't think that going "back" to a `1.0` was the right direction, so I went to `1.1